pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandler.java

```diff
@@ -72,34 +75,34 @@
  * 1. Change compression on raw SV and MV columns.
  * 2. Enabling dictionary on a raw column.
  * 3. Disable forward index on a column where it is enabled.
+ * 4. Enable forward index on a forward index disabled column
```

Comment:
Suggest changing to `"Rebuild forward index for noForwardIndex column"`. Sometimes the enable/disable term with `noForwardIndex` gets confusing because of the nature of the feature.

Reply:
done